### ISSUE TITLE
Rule change proposal: no-duplicate-imports

### ISSUE THREAD

**Issue by geoffliu** [eslint/eslint#8656]
Not sure if this is the right venue to raise this issue, as this proposal involves some Flow-specific syntax. Please redirect if there's a better place to discuss.

What rule do you want to change?
`no-duplicate-imports`

Does this change cause the rule to produce more or fewer warnings?
Fewer

How will the change be implemented? (New option, new default behavior, etc.)?
New default behavior

Please provide some example code that this change will affect:
```js
import type { ... } from 'module'
import { ... } from 'module'
```

What does the rule currently do for this code?
Raises error as there are two imports from the same module, but currently there is no syntax in Flow to import types and values with a single import statement.

What will the rule do after it's changed?
Accepts the example code as valid. Note that this should be a fairly safe change to default behavior, since the only people this change will affect probably aren't using this rule anyway.

**Comment by platinumazure** [comment]
Hi @geoffliu, thanks for the issue!

Since `import type` is experimental syntax, I think we can't accept this enhancement at this time. I would suggest opening an issue in [eslint-plugin-flowtype](https://github.com/gajus/eslint-plugin-flowtype).

**Comment by geoffliu** [comment]
@platinumazure Is the suggestion to have a mirrored `react/no-duplicate-import` and handle this case there?

**Comment by platinumazure** [comment]
@geoffliu Yes, although it would be `flowtype/no-duplicate-import` if I'm not mistaken.

If you use a different ESLint plugin for Flow, then feel free to raise the issue in that plugin instead.

**Comment by geoffliu** [comment]
Ah interesting. I wasn't even aware there's a separate plugin. I was referring to https://github.com/yannickcr/eslint-plugin-react. I'll go bark up that tree then. Cheers!

**Comment by lydell** [comment]
gajus/eslint-plugin-flowtype#72

**Comment by not-an-aardvark** [comment]
Closing because this involves experimental syntax, so we can't support it in core. Feel free to use a plugin like `eslint-plugin-flowtype` for this.
